canisters/src/vts/src/lib.rs

```diff
@@ -110,6 +115,15 @@ pub struct Telemetry {
     pub t_type: TelemetryType,
 }
 
+#[derive(BEncode, BDecode, Debug, PartialEq, Eq, CandidType, Deserialize, Clone, Default)]
```

Comment:
BEncode and BDecode are not needed here, we do not send these types through TCP API.